docs/content/docs/installation/lsp.md

```
@@ -83,14 +83,14 @@ Depending on which LSP plugin you use, configure Emacs to run `cem` for HTML, Ja
 (lsp-register-client
  (make-lsp-client
   :new-connection (lsp-stdio-connection '("cem" "lsp"))
-  :major-modes '(html-mode twig-mode php-mode typescript-mode js-mode)
+  :major-modes '(html-mode twig-mode nunjucks-mode jinja2-mode handlebars-mode php-mode typescript-mode js-mode)
   :server-id 'cem-lsp))
 ```
 
 **eglot:**
 ```elisp
```

markdownlint-cli2 (0.22.1)

[warning] Code fence style
Expected: tilde; Actual: backtick

(MD048, code-fence-style)